src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationProviders.xml

```diff
@@ -2153,6 +2153,27 @@
     <Environment Issuer="https://api.login.yahoo.com/" />
   </Provider>
 
+  <Provider Name="Zendesk" Id="5577b117-1f3d-464f-b987-aabf7c0ab3f7" Documentation="https://developer.zendesk.com/api-reference/introduction/security-and-auth/#oauth-access-token">
```

Comment:
```suggestion
  <Provider Name="Zendesk" Id="5577b117-1f3d-464f-b987-aabf7c0ab3f7" Documentation="https://support.zendesk.com/hc/en-us/articles/4408845965210-Using-OAuth-authentication-with-your-application">
```

Let's use https://support.zendesk.com/hc/en-us/articles/4408845965210-Using-OAuth-authentication-with-your-application instead of https://developer.zendesk.com/api-reference/introduction/security-and-auth/#oauth-access-token as the last one doesn't contain anything really useful.